src/types/index.ts

```diff
@@ -517,3 +517,39 @@ export type CollectDataResultType =
       collectedData?: undefined;
       error: StripeError;
     };
+
+export type LocalMobileUxConfiguration = {
```

Comment:
I think the properties in `LocalMobileUxConfiguration`, `TapZone`, and `Colors` should probably all be nullable? Or some other way to allow users to use defaults. The mapper methods all look like they're set up to handle null values but I think this definition requires them to be specified?

Reply:
yes, that make sense.